File: sample-core/src/main/java/com/quickblox/sample/core/utils/KeyboardUtils.java
package com.quickblox.sample.core.utils;

import android.content.Context;
import android.view.inputmethod.InputMethodManager;
import android.widget.EditText;

import com.quickblox.sample.core.CoreApp;

public class KeyboardUtils {

    public static void showKeyboard(EditText editText) {
        InputMethodManager imm = (InputMethodManager) CoreApp.getInstance().getSystemService(Context.INPUT_METHOD_SERVICE);
        imm.showSoftInput(editText, InputMethodManager.SHOW_IMPLICIT);
    }

    public static void hideKeyboard(EditText editText) {
        InputMethodManager imm = (InputMethodManager) CoreApp.getInstance().getSystemService(Context.INPUT_METHOD_SERVICE);
        imm.hideSoftInputFromWindow(editText.getWindowToken(), 0);
    }

}

File: sample-pushnotifications-kotlin/src/main/java/com/quickblox/sample/pushnotifications/kotlin/utils/KeyboardUtils.kt
package com.quickblox.sample.pushnotifications.kotlin.utils

import android.content.Context
import android.view.inputmethod.InputMethodManager
import android.widget.EditText
import com.quickblox.sample.pushnotifications.kotlin.App

fun showKeyboard(editText: EditText) {
    val imm = App.instanceApp.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
    imm.showSoftInput(editText, InputMethodManager.SHOW_IMPLICIT)
}

fun hideKeyboard(editText: EditText) {
    val imm = App.instanceApp.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
    imm.hideSoftInputFromWindow(editText.windowToken, 0)
}